python/aitemplate/utils/json_utils.py

```diff
@@ -27,7 +29,66 @@
 from aitemplate.compiler.tensor_accessor import TensorAccessor
 
 
+def gen_unique_op_names(sorted_graph: List[Tensor]) -> Dict[Operator, str]:
+    # List is used here, not Set, in order to maintain the order of operators,
+    # depending on memory locations, which may vary from run to run.
+    # Additionally, I don't expect to have too usages for a single op.
```

Comment:
"I don't expect to have too usages" - can you clarify that? many usages? two usages?